101-vm-userdata/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "GEN-UNIQUE"
+    },
+    "userData": {
+      "value": "echo userData"
+    },
+    "dnsLabelPrefix": {
+      "value": "GEN-UNIQUE"
+    },
+    "vmSize": {
+      "value": "Standard_D2_v2"
+    },
+    "ubuntuOSVersion": {
+      "value": "18.04-LTS"
+    },
+    "adminPasswordOrKey": {
+      "value": "GEN-SSH-PUB-KEY"
+    },
+    "location": {
+      "value": "northcentralus"
```

Comment:
if you need to target a specific location you'll need to add a separate param file for USGov: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#cloud-specific-parameter-files

Reply:
This PR will need to wait until at least one gov region has fully deployed in that case, at which point it may just be worth leaving this as a draft until everything is fully deployed. The short term goal here is just to have some example to link to in announcement documentation. We'll probably bundle the template with our code examples as a short term solution until this is ready.